<?xml version="1.0"?>
<launch>
  
  <node name="v2x_simulation1" pkg="v2x_simulation" type="v2x_simulation1" output="screen">              
        <!-- <remap from="AA" to="BB"/> -->
    <!-- <param name="map_origin_lat" value="35.6470671983"/>  -->
    <!-- <param name="map_origin_lon" value="128.401913578"/>  -->
      <param name="map_origin_lat" value="35.64698633754966"/> 
      <param name="map_origin_lon" value="128.40112675332395"/>  
      <!-- <param name="map_origin_att" value="25.797"/>  -->
 
      
  </node>
  
</launch>
